utopia-remix/app/util/api.server.ts

```diff
@@ -79,26 +81,27 @@ async function handleMethod<T>(
     if (validator != null) {
       await validator(request, params)
     }
+
     const resp = await fn(request, params)
     if (resp instanceof Response) {
-      const mergedHeaders = new Headers(defaultResponseHeaders)
+      const headers = new Headers()
```

Comment:
I didn't catch this before, but this should be a `let` to signify that we are mutating it